tools/scripts/purge-folders.sh

```diff
@@ -24,12 +24,12 @@ purge_folders () {
   echo "Purging $1"
   echo "Deleting folders older than $2 days"
 
-  find $1/*/*/ -type d -exec bash -c 'delete_folder "$0" $1' {} $2 \;
+  find $1/*/*/ -type d -prune -exec bash -c 'delete_folder "$0" $1' {} $2 \;
```

Comment:
`-prune` stops the find command from throwing an error after the folder is deleted.